templates/community-viz/README.md

```diff
@@ -65,7 +65,21 @@ You should have two deployments of your visualization: a "dev" version, where
 [caching] is disabled and where you normally develop, and a "prod" version, where
 caching is enabled and you only push "finished" visualizations.
 
-Key commands:
+To deploy:
+
+1. Change `const LOCAL` to `false` in `src/index.js`
+1. Run the appropriate build and push command (see below)
+1. Load your viz in Data Studio
+
+### Key commands:
+
+Update the message and deploy it to your dev bucket:
+
+The `format` argument must be either `object` or `table`. [Read more about formats][ds-component].
+
+```bash
+npm run updateMessage -- --format {object | table}
```

Comment:
Let's just have a separate one for object and table so they can copy and paste this without needing to change it.